package.json

```diff
@@ -47,7 +45,6 @@
     "jest-docblock": "^27.4.0",
     "js-cookie": "^3.0.1",
     "markdown-toc": "^1.2.0",
-    "node-sass": "^9.0.0",
```

Comment:
Don't remove this package, it is used to compile customised sass for our project

Reply:
Ok. So should I revert the changes? It was causing some peer-dependency issues.

Reply:
We need this module, we can't just remove our project's dependency if its causing peer-dependency issues, it would be better if you can find a way around to fix this